ui/src/app/components/common/datagrid/filters/boolean-filter/boolean-filter.component.ts

```diff
@@ -87,6 +76,6 @@ export class BooleanFilterComponent implements ClrDatagridFilterInterface<any>,
   onRemoveFilter() {
     this.isTrueSelected = false;
     this.isFalseSelected = false;
-    this.modelChanges.next([]);
+    this.modelChanges.next();
```

Comment:
You need to call `changes.next()` here. `modelChanges` is not subscribed by anything anymore. `changes` is subscribed by clarity.